save-plugins/warn-plugin/src/commonMain/kotlin/org/cqfn/save/plugin/warn/WarnPlugin.kt

```diff
@@ -195,11 +230,13 @@ class WarnPlugin(
      * @return [TestResult]
      */
     @Suppress("TYPE_ALIAS")
-    internal fun checkResults(expectedWarningsMap: Map<LineColumn?, List<Warning>>,
-                              actualWarningsMap: Map<LineColumn?, List<Warning>>,
```

Comment:
no need to have this complex optimization, we can simply compare lists of warnings